packages/codemod/package.json

```diff
@@ -8,6 +8,7 @@
         "test/dist/**/*",
         "CHANGELOG.md"
     ],
+    "type": "module",
```

Comment:
why is this needed? will there be any downstream complications for users?

Reply:
https://vite.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated has more background. I think it's basically low priority to make the change in any of our dev-facing libraries intended for import, and could cause issues for some folks. For this particular module given intended use it's probably ok, though admittedly deviant standard-wise.

Reply:
Moved back to cjs for consistency, triggered by my finding that ESM required further edits for local iteration running `codemod` binary (now operational, which is nice). We can revisit later.